CommandHelper/LocalPackages/commands/chars/bluebeetlejaimereyes.yml

```diff
@@ -0,0 +1,32 @@
+buffs:
+- /effect <plr> 5 9999 15 true
+- /effect <plr> 11 9999 2 true
+- /effect <plr> 10 9999 2 true
+- /effect <plr> 21 9999 9 true
+- /shot give <plr> blueenergybeam
+- /give <plr> minecraft:elytra
+- /give <plr> minecraft:iron_sword 1 16:10 name:&1B&0lu&1e &0B&1eet&0le&1's &0Bl&1a&0de
```

Comment:
add underscores instead of spaces to the when giving weapons. It should be &1B&0lu&1e_&0B&1eet&0le&1's_&0Bl&1a&0de